src/Console/Controller.php

```diff
@@ -91,6 +108,9 @@ public function actionListen($cwd = null, $timeout = null, $env = [])
         while (true) {
             $this->stdout("Running new process...\n");
             $this->runQueueFetching($command, $cwd, $timeout, $env);
+            if ($this->sleepTimeout) {
```

Comment:
This is not good to use ``isset``. maybe user set timeout to 0 to disable it and ``isset`` always return true for 0 value. with ``if`` we just check timeout is set to greater than 0